### ISSUE TITLE
Dataset for word classigication

### ISSUE THREAD

**Issue by AlibekJ** [Tsabary/fabntt#1]
Hey Tsabary, 

can you get me an array in this (or similar) form: 
```json
[{"word":"ABCD", "length":4, "timeToSkip":300, "output":0},
 {"word":"EFGHI", "length":5, "timeToSkip":0, "output":1}]
```
Where ```output``` codes if word was selected or skipped.

**Comment by Tsabary** [comment]
Yes absolutely. Do we want this array to be individual per user? Or for the
whole system?

‫בתאריך שבת, 11 בינו׳ 2020 ב-4:39 מאת ‪Alibek‬‏ <‪notifications@github.com
‬‏>:‬

**Comment by AlibekJ** [comment]
Would suggest to start with system-wide classificatior first. User-specific LSTM could be added later.

**Comment by Tsabary** [comment]
Saving data with the info we need works just fins now. I can save the words as a String or as an array of the characters. Which one is preferable for you?

Here's how the two ways the data could be saved
https://ibb.co/QXjVFpw

The top expended node is for a shuffled word (so we got the timeToSkip info) and also the word is saved as an array.
The bottom expanded node is for a word that has been saved, so we have no timeToSkip, and the word is saved as a string.

Your thoughts?

**Comment by AlibekJ** [comment]
I think string is better.

**Comment by Tsabary** [comment]
I've refined the code a bit and fixed some things. Had a little issue of a saved word also being added to the ML JSON as a shuffled word, as it was shuffled immediately after being saved. This is fixed.
Also had an issue of duplicates being accepted but that's fixed now.

I have a few more changes I'd like to make in the DB but that shouldn't affect the ML part of things. I'd start playing with it to generate initial database, shouldn't take too long to reach 2000.

Also, I had a little issue with the Git and ended up starting a new repository. This is the new and updated repository

https://github.com/Tsabary/name-generator
